changelog.rst

```diff
@@ -1,3 +1,10 @@
+Upcoming:
+=========
+
+Bug fixes:
+----------
+* Escape switches to VI navigation mode when not canceling completion popup. (Thanks: `Nathan Verzemnieks`_)
```

Comment:
Tests on your PR fail due to the missing link in changelog. Take a look in the bottom of changelog, you'll see lines similar to this:

```
.. _`easteregg`: https://github.com/verfriemelt-dot-org
```

You want to add a line for "Nathan Verzemnieks".

Reply:
Oh! Thanks. I was trying to copy the convention but I didn't actually _think_ about what I was doing ^_^